src/sass/_theme_rst.sass

```diff
@@ -16,22 +16,18 @@
 .rst-content
   // Sphinx by default applies HxW style attributes to images. This fixes that oversite.
   img
+    margin-bottom: $base-line-height
```

Comment:
I'm assuming there will probably be unwanted side effects from this, `img` is a pretty generic selector. Is this on purpose? Looks like the original selector was a bit more specific, at `.rst-content .section > img`